usaspending_api/recipient/models.py

```diff
@@ -137,7 +137,7 @@ class RecipientLookup(models.Model):
     id = models.BigAutoField(primary_key=True)
     recipient_hash = models.UUIDField(unique=True, null=True)
     legal_business_name = models.TextField(null=True, db_index=True)
-    duns = models.TextField(unique=True, null=True)
+    duns = models.TextField(null=True)
```

Comment:
Curious why you removed `unique=True`? Would think if we might want to actually add it to `uei`.

Reply:
Because recipient_lookup uses the hash as the primary key, if we have records that have the same DUNS but different UEI (or one is null and one is not), we can have different records in recipient_lookup with the same DUNS. This happens a lot with transaction records that aren't backfilled vs / SAM records that were updated iirc.